agent/ecscni/netconfig_windows.go

```diff
@@ -31,6 +41,14 @@ func NewVPCENIPluginConfigForTaskNSSetup(eni *eni.ENI, cfg *Config) (*libcni.Net
 		Nameservers: cfg.InstanceENIDNSServerList,
 	}
 
+	// Validate MAC Address, ENI IP Address and ENI Gateway address used for CNI plugin configuration.
+	// Other params are generated at runtime and are considered safe.
```

Comment:
Can we also validate the DNS entries ?

Additionally, would it be more helpful if we could have a uber function to validate the entire network config?

Reply:
That would not be required since they are obtainer from Win32 API call. Therefore, they are considered to be safe sources.